src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx

```diff
@@ -7,11 +7,11 @@ import NEWBIE_SELECT from '../../../graphql/newbieSelect.graphql';
 
 jest.mock('@material-ui/core/Typography', () => 'Typography');
 jest.mock('components/NavBar', () => 'Navbar');
-jest.mock('components/Avatar', () => 'Avatar');
 jest.mock('@material-ui/core/Box', () => 'Box');
 jest.mock('../../PlusButton/', () => 'PlusButton');
 jest.mock('../../Carrousel/', () => 'Carrousel');
 jest.mock('../../BackgroundShape/', () => 'BackgroundShape');
+jest.mock('../../AppWrapper', () => 'AppWrapper');
```

Comment:
Could you update all _new_ imports to use absolute paths, introduced by @Tonik1204 ?
I guess you don't have current changes from master in you PR (probably you will get some conflicts) by anyway take a look at master version of this file:
https://github.com/BuildIt-Poland/buddy-app/blob/2e9e864b3e2f3a876d24cd6aa8d3dd98f170450a/src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx#L14

We still have some places where relative path are being used, so it's good to fix those places when we can 🙂